partcad-cli/src/partcad_cli/click/commands/test.py

```
@@ -0,0 +1,77 @@
+import argparse
+import asyncio
+
+import partcad.logging as pc_logging
+import partcad.utils as pc_utils
+
+
+async def cli_test_async(ctx, packages, sketch, interface, assembly, scene, object):
+    tasks = []
+    for package in packages:
+        if not object is None:
+            if not ":" in object:
```

Ruff (0.8.2)

Test for membership should be `not in`

Convert to `not in`

(E713)